### DISCUSSION TITLE
[DISCUSSION] Appwrite sudo/root mode (gathering users feedback)

### DISCUSSION THREAD

**Discussion by monatis** [appwrite/appwrite#906] [Ideas]
Hi,
I had the chance to test Appwrite, thanks for the great project.
One thing that I can't find is, there are no installation-wide settings on the console. For example, it would be great if we could limit new signups after the first one and control who can create projects on that particular Appwrite installation.

**Comment by eldadfux** [comment]
Hey @monatis , please let me know if this help you:
https://medium.com/appwrite-io/you-can-now-restrict-access-to-your-appwrite-console-b8b447885289

If not, please provide us with more information about your exact use case and needs.

**Comment by monatis** [comment]
hi @eldadfux, yes this serves  the purpose for now, but I would suggest a settings screen on the console similar to self-hosted Gitlab when Appwrite hits stable V1.0. This would make it flawless to manage the whitelist in the long run. For this purpose, the first signed-up user might be deemed to be the root user.

**Comment by eldadfux** [comment]
@monatis Sounds awesome, I will add it to our roadmap for v1.0.0. Would be great if you could bring more feedback on the root user and what features are needed for the global admin dashboard.

**Comment by monatis** [comment]
Hi @eldadfux, thanks a lot for the positive response and adding it to your v1.0 roadmap.

In my opinion and in the ideal world, the root user might be able to:
- activate and/or deactivate front page signups,
- manage whitelists,
- limit the number of projects to be created,
- update the texts of the terms & conditions and privacy policy,
- create "Privacy & Legal" profiles that will be the defaults for future projects,
- grant project creation permission to specific members (e.g., team leaders).

Also, a sudo mode might be implemented, so that the root user need to enter their password again to make root-level changes, etc.

I think that these features will make Appwrite a perfect fit for teams of any size. I hope to contribute with possible commits after I'm done with the ongoing busy period in my projects.
